BuildNotifications.Core/Pipeline/Cache/CacheKey.cs

```diff
@@ -1,8 +1,10 @@
-﻿namespace BuildNotifications.Core.Pipeline.Cache
+﻿using System;
+
+namespace BuildNotifications.Core.Pipeline.Cache
 {
     internal sealed class CacheKey
     {
-        public CacheKey(int providerId, int itemId)
+        public CacheKey(string providerId, string itemId)
```

Comment:
Why string as data type for providerId when everytime it is called it is "Guid.ToString()"?

Reply:
I want to keep the option open for other formats of keys. 
Especially for #48 the GUIDs probably won't work anymore.

I used GUID as a quick & easy fix to get a unique string every time. However I can see how it could be better to expose the Id as string within IProject instead of GUID, so you know that it is a string from the outside?

Reply:
Okay makes sense